test/framework/k8s_utils.go

```diff
@@ -383,3 +488,8 @@ func DeployRootPod(ctx context.Context, c client.Client, namespace string, noden
 	}
 	return &rootPod, nil
 }
+
+// DeployRootPod deploys a pod with root permissions for testing purposes.
+func (*CommonFramework) DeployRootPod(ctx context.Context, c client.Client, namespace string, nodename *string) (*corev1.Pod, error) {
```

Comment:
What is the purpose of this wrapper funciton?

Reply:
removed